### DISCUSSION TITLE
No transient option for table resources ?

### DISCUSSION THREAD

**Discussion by philkpler** [Snowflake-Labs/terraform-provider-snowflake#704] [Q&A]
Hi, 

According to the doc there is no "is_transient" option for the snowflake_table resource, 
whereas it has been implemented for snowflake_schema.
Hence I would have to created a dedicated schema to be able to set my table as transient...

Is that option not available for tables ? 
Would id be possible to add it ?

Thanks,
Philippe

**Comment by RiccardoAmbrosini** [comment]
Is there a solution to this problem?
